frappe/__init__.py

```diff
@@ -530,6 +531,12 @@ def throw(
 	)
 
 
+def delete_folder(path: str) -> None:
+	"""Delete folder and all the content inside that folder."""
+	if os.path.exists(path):
+		shutil.rmtree(path)
```

Comment:
Let's not add more things in `frappe/__init__.py`

We can start a separate FS utils file.

Also this function is same as `shutil.rmtree(path, ignore_errors=True)`